Terminal.Gui/Views/NumericUpDown.cs

```diff
@@ -0,0 +1,224 @@
+            NoDecorations = true,
+            Title = $"{Glyphs.DownArrow}",
+            WantContinuousButtonPressed = true,
+            CanFocus = false,
+            ShadowStyle = ShadowStyle.None
+        };
+
+        _number = new ()
+        {
+            Text = Value?.ToString () ?? "Err",
+            X = Pos.Right (_down),
+            Y = Pos.Top (_down),
+            Width = Dim.Auto (minimumContentDim: Dim.Func (() => string.Format (Format, Value).Length)),
+            Height = 1,
+            TextAlignment = Alignment.Center,
+            CanFocus = true
+        };
+
+        _up = new ()
+        {
+            X = Pos.Right (_number),
+            Y = Pos.Top (_number),
+            Height = 1,
+            Width = 1,
+            NoPadding = true,
+            NoDecorations = true,
+            Title = $"{Glyphs.UpArrow}",
+            WantContinuousButtonPressed = true,
+            CanFocus = false,
+            ShadowStyle = ShadowStyle.None
+        };
+
+        CanFocus = true;
+
+        _down.Accept += OnDownButtonOnAccept;
+        _up.Accept += OnUpButtonOnAccept;
+
+        Add (_down, _number, _up);
+
+        AddCommand (
+                    Command.ScrollUp,
+                    () =>
+                    {
+                        if (type == typeof (object))
+                        {
+                            return false;
+                        }
+
+                        if (Value is { })
+                        {
+                            Value = (dynamic)Value + (dynamic)Increment;
+                        }
+
+                        return true;
+                    });
+
+        AddCommand (
+                    Command.ScrollDown,
+                    () =>
+                    {
+                        if (type == typeof (object))
+                        {
+                            return false;
+                        }
+
+                        if (Value is { })
+                        {
+                            Value = (dynamic)Value - (dynamic)Increment;
+                        }
+
+                        return true;
+                    });
+
+        KeyBindings.Add (Key.CursorUp, Command.ScrollUp);
+        KeyBindings.Add (Key.CursorDown, Command.ScrollDown);
+
+        SetText ();
+
+        return;
+
+        void OnDownButtonOnAccept (object? s, HandledEventArgs e) { InvokeCommand (Command.ScrollDown); }
+
+        void OnUpButtonOnAccept (object? s, HandledEventArgs e) { InvokeCommand (Command.ScrollUp); }
+    }
+
+    private T _value = default!;
+
+    /// <summary>
+    ///     Gets or sets the value that will be incremented or decremented.
+    /// </summary>
+    /// <remarks>
+    ///     <para>
+    ///         <see cref="ValueChanging"/> and <see cref="ValueChanged"/> events are raised when the value changes.
+    ///         The <see cref="ValueChanging"/> event can be canceled the change setting <see cref="CancelEventArgs{T}.Cancel"/> to
```

Comment:
Remove the `Cancel` property because it's only exposed an a instance.

Reply:
Not sure what you mean.

Reply:
The cref="CancelEventArgs{T}.Cancel"/> is generation a warning and remove it the warning disappear.